packages/server/src/tests/epics/momentumTest.ts

```diff
@@ -26,17 +27,40 @@ const jumps = {
     high: 5,
 } as const;
 
-function isIncreasedEngagement(weeksInWindow: WeeklyArticleHistory): boolean {
-    const categoryForThirdMonth = getCategoryOfArticleViewed(weeksInWindow.slice(0, 4));
-    const categoryForSecondMonth = getCategoryOfArticleViewed(weeksInWindow.slice(4, 8));
-    const categoryForFirstMonth = getCategoryOfArticleViewed(weeksInWindow.slice(8));
-    if (
-        categoryForThirdMonth >= categoryForSecondMonth &&
-        categoryForSecondMonth >= categoryForFirstMonth
-    ) {
-        return categoryForThirdMonth - categoryForFirstMonth >= jumps.mediumHigh;
-    }
-    return false;
+export function getThreeMonthsHistory(
+    articleHistory: WeeklyArticleHistory,
+    now: Date = new Date(),
+) {
+    const mostRecentMonthHistory = getWeeksInWindow(articleHistory, 4);
+
+    const startOfSecondMostRecentMonth = subWeeks(now, 4);
+
+    const secondMostRecentMonthHistory = getWeeksInWindow(
+        articleHistory,
+        4,
+        startOfSecondMostRecentMonth,
+    ).slice(1); // remove overlapping week
```

Comment:
an alternative to this would be to change `getWeeksInWindow` to take an extra boolean, e.g. `includePartialWeek`, with a nice comment explaining what it does!